compiler/rustc_expand/src/config.rs

```diff
@@ -445,7 +445,7 @@ impl<'a> StripUnconfigured<'a> {
             }
         };
         parse_cfg(&meta_item, &self.sess).map_or(true, |meta_item| {
-            attr::cfg_matches(&meta_item, &self.sess.parse_sess, self.features)
+            attr::cfg_matches(&meta_item, &self.sess.parse_sess, None, self.features)
```

Comment:
I strongly suspect that `cx.current_expansion.lint_node_id` can be passed to both `cfg_matches` calls in this file.

Reply:
It can, but gives `CRATE_NODE_ID` (not very helpful).

Reply:
Done.

Reply:
It should only give `CRATE_NODE_ID` if `CRATE_NODE_ID` is the closest "stable"/"already expanded" node, otherwise it should give closer nodes.